src/InteractiveWindow/Version.props

```diff
@@ -1,5 +1,5 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <RoslynSemanticVersion>1.2.0</RoslynSemanticVersion>
+    <RoslynSemanticVersion>2.0.0</RoslynSemanticVersion>
```

Comment:
I think that's possible, but we'd have to opt out of a bunch of binding redirect helpers.  Would it be alright to make that a separate issue - presumably against master?